packages/react-native-jigsaw/src/components/Text.tsx

```diff
@@ -56,20 +56,20 @@ export const SEED_DATA = {
     children: {
       group: GROUPS.data,
       label: "Text",
-      description: "Text for the text",
+      description: "Text",
       editable: true,
       required: true,
       formType: FORM_TYPES.string,
       propType: PROP_TYPES.STRING,
-      defaultValue: "Change me under the Data Panel",
```

Comment:
I don't recall seeing this anywhere, I wonder if this was even used?

Reply:
never was used b/c of a change on the server lol